src/Google2FA.php

```diff
@@ -231,7 +231,7 @@ private function makeStartingTimestamp($window, $timestamp, $oldTimestamp = null
     {
         return is_null($oldTimestamp)
             ? $timestamp - $this->getWindow($window)
-            : max($timestamp - $this->getWindow($window), $oldTimestamp + 1);
+            : max($timestamp - $this->getWindow($window), intval($oldTimestamp) + 1);
```

Comment:
It's documented to only accept `int`: can you trace why you have something else?

Do you have a `string`?

Reply:
the error `Unsupported operand types: string + int `  start coming after the recent update so using `intval` fix the problem  incase number come in string

Reply:
I understand, but the whole code base documents it as `int`, so why are you having a string?

If the library moves to types, this would have to be removed because then it's not possible to have non-ints pass in so I asked, why do you have `string` in the first place?